.idea/workspace.xml

```diff
@@ -12,14 +12,15 @@
   <component name="Git.Settings">
```

Comment:
could we add this .idea folder to .gitignore?

Reply:
And all the .idea/ files

Reply:
removed in this commit: 2e5aa9a953f625eb2cd59737a2a3b2100a455d1a